card_parser.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'flashcard'
+require 'pry'
+module Parsable
+
+ def parse_from_txt(file_path)
+    flashcard_objects = []
+    txt_lines = File.read(file_path).split(/\n/)
```

Comment:
Rather than splitting here, you could use the `File.readlines` method which gives you an array of lines split on new lines.